Dialog.php

```diff
@@ -39,6 +39,13 @@ public function init()
     {
         parent::init();
         echo Html::beginTag('div', $this->options) . "\n";
+        
+        //Temporary fix of closing icon (x) not showing up in dialog
```

Comment:
I don't think it will be temporary so let's the comment be "Fix for closing icon (x) not showing up in dialog"